.travis.yml

```diff
@@ -127,6 +127,20 @@ matrix:
         - memcached
         - redis-server
         - postgresql
+    - python: 2.7
+      env: TEST_SUITE=snuba FLASK_DEBUG=1 CLICKHOUSE_TABLE=test SNUBA=http://localhost:8000
+      services:
+        - docker
+        - memcached
+        - redis-server
+        - postgresql
+      before_install:
+        - docker run -d --name clickhouse-server -p 9000:9000 -p 9009:9009 -p 8123:8123 --ulimit nofile=262144:262144 yandex/clickhouse-server
+        - docker run -d --env FLASK_DEBUG=1 --env CLICKHOUSE_TABLE=test --name snuba -p 8000:8000 --link clickhouse-server:clickhouse-server getsentry/snuba
```

Comment:
is it cleaner to have these containers running on a named bridge network rather than mappping/exposing the clickhouse ips and providing the `--link` ? I suppose the effect is in the end the same.

Reply:
I do like Snuba to Clickhouse, but the tests themselves don't run in Docker. Would a named bridge help the tests find and hit the Snuba service?